proto/buf/alpha/lint/v1/config.proto

```diff
@@ -41,6 +41,8 @@ message Config {
   // this value with the given string.
   string enum_zero_value_suffix = 6;
   // rpc_allow_same_request_response allows the same message type for both the request and response of an RPC.
+  bool rpc_allow_same_request_response = 11;
```

Comment:
We haven't actually used this over the wire yet, so I'd prefer to reorder the field numbers so that this actually fits into the mold (i.e. this would be `7`, then the following fields would all be incremented by 1).

Reply:
I would, but I ran into the `buf breaking` check in our lint. It appears we have started to enforce the breaking change detection? This was something I wanted to discuss, haha.

Reply:
It's because we have a `v1` package name here - we only protect against `v1alpha1`. But this is a new package and it's unreleased, so feel free to break it this one time.

Reply:
Ah yeah -- urgh, lol. Okay, sounds good, since I have your written approval here, I will push up that change :)